pkg/asset/templates/kubelet.yaml

```diff
@@ -27,17 +27,11 @@ spec:
         - --api-servers={{ index .APIServers 0 }}
         - --config=/etc/kubernetes/manifests
         - --allow-privileged
-        - --hostname-override=$(MY_POD_IP)
```

Comment:
Could we carry that in the CoreOS Hyperkube and update bootkube to make use of it?